src/cli/commands/diff.py

```
@@ -0,0 +1,108 @@
+def run(args: dict) -> None:
+    file_a = args.get("file_a")
+    file_b = args.get("file_b")
+
+    from src.core.parser import BaseParser as _BP
+    from src.core.ast_normalizer import _extract_name as _en
+
+    class _DummyParser(_BP):
+        def parse_file(self, fp: str) -> ParseResult:
+            import subprocess, json, tempfile, os
+            parse_bin = "./parse"
+            r = subprocess.run([parse_bin, "-file", fp], capture_output=True, timeout=30)
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)